lib/rouge/demos/lutin

```diff
@@ -0,0 +1,84 @@
+-- The examples provided in [SIES'13] "Engineering Functional
```

Comment:
It'd be good to trim down the demos/ files a bit. Most are in the 10-30 line length, as they're meant to be able to give a tiny showcase such as e.g. in the boxes here: http://rouge.jneen.net/

Reply:
Sure, no pb. There are 2 sets of demos :
- lib/rouge/demos/
- spec/visual/samples/

Do you want me to reduce both?

Reply:
Ok. So I will probably leave it as it is, so that the example compiles.

Thanks for taking care of this.